pages/_app.js

```diff
@@ -108,6 +108,9 @@ class Root extends App {
       ReactGA.initialize(GA_ID, { debug: false });
       ReactGA.ga('send', 'pageview', url, { hitCallback: clearUtmParams });
```

Comment:
do we also need to keep `clearUtmParams` here?

Reply:
Yep